export.js

```diff
@@ -3075,7 +3075,7 @@ if ( !AmCharts.translations[ "export" ][ "en" ] ) {
 				_this.handleCallback( callback, "data", false );
 
 				// READY CALLBACK FOR EACH DEPENDENCY
-				for ( filename in _this.libs.namespaces ) {
+				for ( const filename in _this.libs.namespaces ) {

```

Comment:
Could you change `const` to `var`? export.js uses ES5. `const` is only available for ES2015 +.